Assignment/traning4/src/task1/Task1.js

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useState } from 'react';
+import axios from 'axios';
+import './Task1.css';
+
+
+const callJoke = async () => {
+    axios
+        .get('https://github.com/15Dkatz/official_joke_api/blob/master/jokes/index.json')
+        .then(function (response) {
+            console.log(response);
+        })
+        .catch(function (error) {
+            console.log(error);
+        });
+}
+
+
+function Task1() {
+
+    const typingTimeoutRef = useRef(null);
+    const [joke, setJoke] = useState([]);
+
+
+    useEffect(() => {
+        async function fetchData()
+        {
+            try{
+                const requestUrl = `https://github.com/15Dkatz/official_joke_api`;
+                const response = await fetch(requestUrl);
+                const responseJSON = await response.json();
+                setListCountry(responseJSON);    
+            }
+            catch(error)
+            {
+                console.log(error);
+            }
+        }
+        fetchData();
+    }, []);
+
+
+    const handleGetJoke = async (e) =>
```

Comment:
@lehuy1012 why we need async function here

Reply:
i was  foreget  clean async